src/offline.js

```diff
@@ -136,6 +139,11 @@ export function replicateFromCozy (cozy, doctype, options = {}) {
       return reject(new Error('You can\'t use `live` option with Cozy couchdb.'))
     }
 
+    if (isOffline()) {
+      reject(replicationOfflineError)
+      return options.onError && options.onError(replicationOfflineError)
```

Comment:
In a promise function, calling `resolve` or `reject` won't let subsequent code to be executed. In other words, the instruction `return options.onError && options.onError(replicationOfflineError)` won't be executed, never.